src/Data/Event/LocalStorage/localStorageEventDataClass.ts

```diff
@@ -90,6 +90,13 @@ async function attributes(): Promise<DataClassSchema> {
       'string',
       { title: lang === 'de' ? 'Veranstalter' : 'Organizer' },
     ],
+    responsibleAgent: [
+      'reference',
+      {
+        title: lang === 'de' ? 'Zuständiger Bearbeiter' : 'Responsible agent',
```

Comment:
* [ ] FYI the wording is not helpful in determining which role this person plays in an event. I'd prefer something like Event coordinator/Veranstaltungsleiter.